### DISCUSSION TITLE
How to enable VERBOSE logging?

### DISCUSSION THREAD

**Discussion by dllu** [cdcseacave/openMVS#1004] [General]
When running DensifyPointCloud or InterfaceCOLMAP, it sometimes just quits with exit code -1 without printing out anything or explaining why. For example, when running `InterfaceCOLMAP`, it just says 
```
Reading points: /home/dllu/my_data/sparse/points3D.txt
```
and immediately quits with exit code -1.

In another instance, when running DensifyPointCloud, it also quits with exit code -1 after printing out

```
Geometric-consistent estimated depth-maps 4308 (100%, 1m42s946ms)
```

I looked through `BUILD.md` and couldn't see any documentation on verbosity, and https://github.com/cdcseacave/openMVS/blob/master/libs/Common/Log.h and https://github.com/cdcseacave/openMVS/blob/master/libs/Common/Common.h have rather complex macros which I don't understand.

Thanks!

**Comment by cdcseacave** [comment]
all binaries in openmvs support the `-v` param, try to set it to a higher value, ex `-v5` and see if you get what you need
otherwise use debug mode
